core/server/src/test/java/alluxio/master/MasterSourceTest.java

```diff
@@ -284,9 +285,12 @@ public void completeFileTest() throws Exception {
    */
   @Test
   public void deletePathTest() throws Exception {
-
     // cannot delete root
-    Assert.assertFalse(mFileSystemMaster.delete(ROOT_URI, true));
+    try {
+      mFileSystemMaster.delete(ROOT_URI, true);
+    } catch (InvalidPathException e) {
```

Comment:
This unit test is for `MasterSource`, not `FilesystemMaster`, so I want to keep the test focused on `MasterSource`.